frontend/src/services/songs.ts

```diff
@@ -17,6 +33,48 @@ export async function nextSongbookSong(sessionKey: string | undefined) {
   }
 }
 
+//start of createSongbook
```

Comment:
I assume these comments were for while you were working on it? If so let's remove, they're not adding anything imo

Reply:
Deleted.